recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,756 @@
+                , "freetype":          "freetype/2.11.1" # ... what? ... WORKED with 2022-05-06-freetype
+                , "glew":              "glew/2.2.0"
+                , "jsoncpp":           "jsoncpp/1.9.5"
+                # , "libharu": "libharu/2.3.0" -- use VTK's bundled version - heavily patched
+                , "lz4":               "lz4/1.9.3"
+                , "proj":              "proj/9.0.0" # if MAJOR version changes, update ThirdParty/libproj/CMakeLists.txt
+                , "pugixml":           "pugixml/1.12.1"
+                , "sqlite3":           "sqlite3/3.38.1"
+                , "utfcpp":            "utfcpp/3.2.1"
+                , "xz_utils":          "xz_utils/5.2.5" # VTK calls this lzma
+                }
+
+        # TODO figure out how we want to support modules...
+        # VTK already has an extensive module system, we would want to mirror or use it
+        # if self.options.module_hdf5:
+        # parties["hdf5"] = "hdf5/1.12.1"
+
+        if self.options.qt:
+            parties["qt"] = "qt/6.2.4"
+
+            # NOTE: could also try QT's offical QT
+            # self.requires("qtbase/6.2.4@qt/everywhere")
+
+        return parties
+
+
+    def requirements(self):
+        for pack in self._third_party().values():
+            self.requires(pack)
+
+
+    def build_requirements(self):
+        if self.options.rendering:
+            Apt(self).install([
+                "build-essential",
+                "mesa-common-dev",
+                "mesa-utils",
+                "freeglut3-dev"
+
+                # These were listed by the previous recipe author,
+                # but they are not listed in vtk Documentation/dev/build.md
+                # so I'm not sure if they are really needed ... TODO
+                #
+                # "mesa-utils-extra",
+                # "libgl1-mesa-dev",
+                # "libglapi-mesa",
+                # "libsm-dev",
+                # "libx11-dev",
+                # "libxext-dev",
+                # "libxt-dev",
+                # "libglu1-mesa-dev"
+                ])
+            # TODO people on other platforms please
+            # contribute what packages are required
+
+
+    def validate(self):
+        if not self.options.shared and self.options.enable_kits:
+            raise ConanInvalidConfiguration("KITS can only be enabled with shared")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        # 64 / 32 bit IDs
+        if self.options.use_64bit_ids != "Auto":
+            tc.variables["VTK_USE_64BIT_IDS"] = self.options.use_64bit_ids
+
+        # Be sure to set this, otherwise vtkCompilerChecks.cmake will downgrade our CXX standard to 11
+        tc.variables["VTK_IGNORE_CMAKE_CXX11_CHECKS"] = "ON"
```

Comment:
I think you may use just True/False instead of "ON"/"OFF"

Reply:
Ok will give that a try